tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -221,6 +221,13 @@ <h2>Continue...</h2>
         max-width: 540px;
         margin: 80px auto 0 auto;
       }
+      .sidebar {
+        position: relative;
+        height: 100%;
+        /* width: 40%; */
```

Comment:
We can probably just delete these commented lines.

Reply:
Done.